src/main/resources/logback.xml

```diff
@@ -26,8 +42,16 @@
     </appender>
 
     <root level="DEBUG">
-        <appender-ref ref="STDOUT" />
+        <if condition='p("ASJSON").contains("true")'>
+            <then>
+                <appender-ref ref="STASH" />
+            </then>
+            <else>
+                <appender-ref ref="STDOUT" />
+            </else>
+        </if>
         <appender-ref ref="FILE" />
+        <appender-ref ref="METRICS" />
```

Comment:
should be included now?

Reply:
Not! I did remove appender but forgot to remove ref to it